addons/road-generator/nodes/road_lane.gd

```diff
@@ -101,18 +101,19 @@ func register_vehicle(vehicle: Node) -> void:
 ## Optional but good cleanup of references.
 func unregister_vehicle(vehicle: Node) -> void:
 	if vehicle in _vehicles_in_lane:
+		print("Unregistering vehicle.")
```

Comment:
Either remove this, or gate it behind a new class level member variable called "debug: bool = false" at the top